README.md

```diff
@@ -1,8 +1,251 @@
+
+* Install FuseML and fuseml-core client
+
+  Follow the [FuseML installation guide](https://github.com/fuseml/fuseml#usage) to install all necessary services, including `fuseml-core`.
+
+  Install `fuseml_core-cli` binary to some place within your PATH so you do not need to execute it with the full path.
+
+  Make sure to use correct `-url` value for `fuseml-core` service for *each fuseml_core-cli command*. Remember, get the right URL with
+
+  ```bash
+  kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}"
+  ```
+  
+
+* Get the example code
+
+  Check out the [examples](https://github.com/fuseml/examples) project:
+
+  ```bash
+  git clone git@github.com:fuseml/examples.git fuseml-examples
+  cd fuseml-examples
+  ```
+
+  Under `models/mlflow-wines` directory there is the example MLflow project. It's only slightly modified example based on the [upstream MLflow](https://github.com/mlflow/mlflow/tree/master/examples) one.
+
+  Under `pipelines` directory there is an example of FuseML workflow definition.
+
+* Register the codeset
+
+  Register the example MLflow model as a codeset:
+  ```bash
+  fuseml_core-cli codeset register -name "mlflow-test" --project "mlflow-project-01" --location "models/mlflow-wines"
+  ```
+
+* Update the example to fit your setup
+
+  The [workflow definition example](https://github.com/fuseml/examples/blob/main/pipelines/pipeline-01.yaml) has some hardcoded values that need to be changed for your specific environment. Namely, see the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` values: these are the credentials to the S3 based minio store that was installed to your cluster by `fuseml-installer`.
+
+  To get these values from your cluster setup, run
+  ```bash
+  export ACCESS=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json| jq -r '.["data"]["accesskey"]' | base64 -d)
+  export SECRET=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json| jq -r '.["data"]["secretkey"]' | base64 -d)
+  ```
+
+  Now replace the original values in the pipeline-01.yaml example. You can do it by editing the file manually or by running following command:
+  ```bash
+  sed -i -e "/AWS_ACCESS_KEY_ID/{N;s/value: [^ \t]*/value: $ACCESS/}" pipelines/pipeline-01.yaml
+  sed -i -e "/AWS_SECRET_ACCESS_KEY/{N;s/value: [^ \t]*/value: $SECRET/}" pipelines/pipeline-01.yaml
+  ```
+
+* Create a workflow
+
+  Use the modified example workflow definition:a
+
+  FIXME use YaML!!!
+  ```bash
+  workflow=$(cat pipelines/pipeline-01.json)
+  fuseml_core-cli workflow register --body "$wf"
+  ```
+
+* Assign the codeset to workflow
+
+  ```bash
+  fuseml_core-cli workflow assign -name mlflow-sklearn-e2e -codeset-name mlflow-test -codeset-project mlflow-project-01
+  ```
+
+  Now that the Worfklow got specific Codeset assigned, it initiated workflow run. 
+  Browse to `TEKTON_DASHBOARD_URL` and click PipelineRuns. Once the run is succeeded, new application will occur.
```

Comment:
or use `workflow list-runs`?

I would also change "new application will occur" to "a new FuseML application will be created".